src/test/miner_tests.cpp

```diff
@@ -776,6 +788,52 @@ BOOST_AUTO_TEST_CASE(CreateNewBlock_validity) {
     fCheckpointsEnabled = true;
 }
 
+// NOTE: These tests rely on CreateNewBlock doing its own self-validation!
+BOOST_AUTO_TEST_CASE(CreateNewBlock_minerfund) {
+    // Note that by default, these tests run with size accounting enabled.
+    GlobalConfig config;
+    // Don't check this functionality in this test.
+    config.SetEnableMinerFund(true);
+    const CChainParams &chainparams = config.GetChainParams();
+    CScript scriptPubKey =
+        CScript() << ParseHex("04678afdb0fe5548271967f1a67130b7105cd6a828e03909"
+                              "a67962e0ea1f61deb649f6bc3f4cef38c4f35504e51ec112"
+                              "de5c384df7ba0b8d578a4c702b6bf11d5f")
+                  << OP_CHECKSIG;
+    std::unique_ptr<CBlockTemplate> pblocktemplate;
+    CMutableTransaction tx;
+    CScript script;
+    TestMemPoolEntryHelper entry;
+    entry.nFee = 11 * SATOSHI;
+    entry.nHeight = 11;
```

Comment:
Doesn't seem to be used? Please check if you can simplify this test.

Reply:
Yeah, I'll look through. Probably missed some copypasta.